electron/src/main.ts

```diff
@@ -203,6 +220,10 @@ const showMainWindow = (mainWindowState: WindowStateKeeper.State) => {
     main.webContents.openDevTools({mode: 'detach'});
   }
 
+  if (session.defaultSession) {
+    session.defaultSession.allowNTLMCredentialsForDomains('*');
```

Comment:
The origin of the proxy should be enough, otherwise we open this to all domains. If it's not available, maybe then we should enable it only if a proxy as been set.